__tests__/utils.test.ts

```diff
@@ -87,7 +84,7 @@ describe('Test secret value retrieval', () => {
     });
 
     test('Throws an error if unable to retrieve the secret', async () => {
-        const error = new ResourceNotFoundException({$metadata: {} as any, Message: 'Error'});
+        const error = new ResourceNotFoundException({$metadata: {}, message: 'Error'});
```

Comment:
So are changes like this done by some linter, or done by you?

Is this just scope creep?

Reply:
Fixes build failure: https://github.com/aws-actions/aws-secretsmanager-get-secrets/actions/runs/5308032009/jobs/9607113113